content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -27,11 +27,11 @@ multiple scheduling retries will be needed.
 
 ## {{% heading "prerequisites" %}}
 
-Kubernetes v{{< skew currentVersion >}} includes cluster-level API support for
+Kubernetes v{{< skew currentPatchVersion >}} includes cluster-level API support for
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.